.agents/skills/wpilog-decode/scripts/wpilog_common.py

```
@@ -0,0 +1,39 @@
+import re
+from pathlib import Path
+
+
+def _version_key(name: str) -> tuple[int, ...]:
+    """Sort WPILib artifact versions, ranking beta < rc < stable."""
+    key: list[int] = []
+    for part in name.split("."):
+        for token in re.split(r"[-_]", part):
+            if token.isdigit():
+                key.append(int(token))
+            elif token == "beta":
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token"

(S105)